src/main.py

```diff
@@ -30,7 +23,7 @@ def signal_handler(sig, frame):
 # b2_player = Player('b',n_b2)
 
 tournament=None
-self_play_batch_size=10000
+self_play_batch_size=5000
```

Comment:
I am all for shortening training time, but this does mean that we cannot compare against Dawson's original results. We will need to reproduce them ourselves.
Given how long it's taking to do this for me, it might end up saving us more time to keep this at 10000 and just refer to the original results for comparison.

Reply:
Resolved in chat